backend/open_webui/retrieval/utils.py

```
@@ -11,10 +11,17 @@
 from langchain_community.retrievers import BM25Retriever
 from langchain_core.documents import Document
 
+
+from open_webui.config import VECTOR_DB
 from open_webui.retrieval.vector.connector import VECTOR_DB_CLIENT
 from open_webui.utils.misc import get_last_user_message
```

Ruff (0.8.2)

`open_webui.utils.misc.get_last_user_message` imported but unused

Remove unused import: `open_webui.utils.misc.get_last_user_message`

(F401)